CHANGELOG.rst

```diff
@@ -34,6 +34,8 @@ Changelog
 * Added a check to :class:`~cryptography.x509.NameConstraints` to ensure that
   :class:`~cryptography.x509.DNSName` constraints do not contain any ``*``
   wildcards.
+* Removed many unused CFFI OpenSSL bindings. This will not impact you unless
+  you are using ``cryptography`` to directly invoke OpenSSL's C API.
```

Comment:
I feel like we should mention that our bindings aren't public API 😄

Reply:
Meaning by not documenting these changes, or by including that note here?

Reply:
Including the note here. I am in favor of saying we made changes but also that this isn’t supported surface area.

Reply:
Maybe we say come talk about adding features to cryptography if you need some functionality you’re rummaging around in the bindings to get right now. 🤷‍♀️

Reply:
Decided not to add the note, I don't want to be perceived as promising we'll add them, when we know some external users want to do things that are bad.